app/jobs/regular/notify_reviewable.rb

```diff
@@ -30,41 +23,63 @@ def execute(args)
       end
     end
 
-    # admins
-    notify(
-      User.real.admins.pluck(:id),
+    counts = Hash.new(0)
```

Comment:
```suggestion
    counts = {}
```

Reply:
Can you share why? There are a few places where I do `count[:something] += 1` and without defaulting the hash to zero I'd have to do `count[:something] = (count[:something] || 0) + 1` which is slightly annoying.

Reply:
Oops I missed the `0` xD. Ignore my comment here.